dm/ctl/common/util.go

```diff
@@ -32,9 +32,16 @@ import (
 
 var (
 	masterClient pb.MasterClient
+	globalConfig *Config
```

Comment:
should we make a default global config, thus `GlobalConfig()` would never return a nil